apps/judicial-system/api/src/app/modules/case/case.pdf.ts

```diff
@@ -73,19 +90,18 @@ export async function generateRulingPdf(existingCase: Case): Promise<string> {
   const html = compiledFunction({
     courtCaseNumber: existingCase.courtCaseNumber,
     policeCaseNumber: existingCase.policeCaseNumber,
-    courtStartTime: formatTime(existingCase.courtStartTime),
-    courtEndTime: formatTime(existingCase.courtEndTime),
-    courtDate: formatDate(existingCase.courtStartTime),
+    courtStartTime: formatDate(existingCase.courtStartTime, 'HH:mm'),
```

Comment:
We have a constant for time format in Constants 👉  https://github.com/island-is/island.is/blob/master/apps/judicial-system/web/src/utils/constants.ts#L13

Consider using that instead